vaadin-text-field-flow-parent/vaadin-text-field-flow/src/main/java/com/vaadin/flow/component/textfield/AbstractNumberField.java

```diff
@@ -142,16 +143,12 @@ public void setErrorMessage(String errorMessage) {
         super.setErrorMessage(errorMessage);
     }
 
-    @Override
-    public void setLabel(String label) {
-        super.setLabel(label);
-    }
```

Comment:
This method was not removed in the original PR. Doing so might break binary compatibility, so let's keep it.